source/api/commands/http.md

```diff
@@ -469,9 +469,14 @@ The available functions on `res` are:
 }
 ```
 
+{% history %}
+{% url "6.0.0" changelog#6-0-0 %} | Renamed `cy.route2()` to `cy.http()`.
+{% url "5.1.0" changelog#5-1-0 %} | Added experimental `cy.route2()` command under `experimentalNetworkStubbing` option.
+{% endhistory %}
+
 # See also
 
 * {% url "`cy.route2()` example recipes with real-world examples" https://github.com/cypress-io/cypress-example-recipes#stubbing-and-spying %}
-* {% url "`cy.route` vs `cy.route2`" https://glebbahmutov.com/blog/cy-route-vs-route2/ %} blog post
+* {% url "`cy.route()` vs `cy.route2()`" https://glebbahmutov.com/blog/cy-route-vs-route2/ %} blog post
```

Comment:
These links will need to be updated to newer http recipe + blog. Not blocking this PR necessarily.

Reply:
yep, i was thinking incremental updates here